plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoConnectorTest.java

```diff
@@ -380,4 +380,61 @@ private void assertOneNotNullResult(String query)
         assertEquals(results.getMaterializedRows().get(0).getFieldCount(), 1);
         assertNotNull(results.getMaterializedRows().get(0).getField(0));
     }
+
+    /*
+     * Content Merge from BaseMongoDistributedQueries.java
+     * supportsDelete() supportsViews() supportsCommentOnTable() supportsCommentOnColumn() not include
+     * These functions are already final, can't be override
+     */
+
+    public void testCreateSchema()
+    {
+        // the connector does not support creating schemas
+    }
+
+    public void testRenameTable()
+    {
+        // the connector does not support renaming tables
```

Comment:
Instead of overriding the test implement the `hasBehavior` method

```
@Override
protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
{
    switch (connectorBehavior) {
        case SUPPORTS_RENAME_TABLE:
            return false;

        ,,,

        default:
            return super.hasBehavior(connectorBehavior);
    }
}
```

Reply:
Within `hasBehavior`, please adhere to the order of options as they are defined in `TestingConnectorBehavior`. This will make it easier to read.